recipes/simple-pid/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "simple-pid" %}
+{% set version = "1.0.1" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 60d6b3a7c3f19734d33bbc5938293c4924d7cfea74176def03306057b3f99524
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - openpyxl >=3
+    - pandas
+    - py4xs
```

Comment:
Why do we need beamline-specific packages here? It's a general-purpose package.

Here are the original dependencies: https://github.com/nsls-ii-forge/simple-pid-feedstock/blob/ab6a12b3d6bd0898c23b781ae00945dd9b76e5ce/recipe/meta.yaml#L18-L23.

Reply:
In the commands: section of the meta.yaml there are the import lines. It was giving a " no module name: "pkg_resourse" " error and so I compared it to [lixtools](https://github.com/conda-forge/staged-recipes/pull/15648/files) meta.yaml which uses the same type of import command and just put tested different packages from it till it passed all the checks. 

I put in openpyxl, pandas, and py4xs together so I'm not sure which one made it work.

Reply:
See https://github.com/nsls-ii-forge/pyfiglet-feedstock/blob/d92d0710d3dfdd97120b1c0c12a061f8c341b33d/recipe/meta.yaml#L26-L27 as an example. You'll need to add `setuptools` to the run dependencies to provide `pkg_resources` lib.

Reply:
Since it's mostly related to tests, let's add it to the `test` block's dependencies instead.